src/test/kotlin/logic/feature/projectHistoryHandling/createLogHelper.kt

```diff
@@ -0,0 +1,21 @@
+package logic.feature.projectHistoryHandling
+
+import kotlinx.datetime.LocalDateTime
+import org.damascus.logic.entities.ActionLog
+import java.util.UUID
+
+fun createFakeActionLog(
+    userName: String = "TestMate",
+    taskId: UUID,
+    currentState: String = "TODO",
+    targetedState: String = "In-progress",
```

Comment:
Please merge with develop and utilize the State model instead of String for the states.

Reply:
That's not what I meant, it should be

```
currentState: State
targetedState: State
```

Also, you can use a better name for `targeted` like `new` to better reflect the meaning of the value.